### ISSUE TITLE
issues with ruby tutorial

### ISSUE THREAD

**Issue by ConorSheehan1** [howtographql/howtographql#891]
re pr https://github.com/howtographql/howtographql/pull/890

I think I found a few more issues.

1. The `resolve` function in [3-mutations.md](https://github.com/howtographql/howtographql/blob/master/content/backend/graphql-ruby/3-mutations.md) throws and exception because it's missing an `end` keyword.

2. `rails generate graphql:install` seems to not create DateTimeType (used in [2-queries.md](https://github.com/howtographql/howtographql/blob/master/content/backend/graphql-ruby/2-queries.md)) or BaseMutations (used in [3-mutations.md](https://github.com/howtographql/howtographql/blob/master/content/backend/graphql-ruby/3-mutations.md)). Related issue https://github.com/howtographql/graphql-ruby/issues/20

I have the resolve error sorted already https://github.com/ConorSheehan1/howtographql/commit/8111e7ed098b4394b2a34d6a9a66b2d94f101ade, but I'm trying to figure out if the `DateTimeType` and `BaseMutations` issues could be down to gem versions or my environment.

Would you prefer if I wait and add all of those changes as one pr?
Or add them as separate prs?

**Comment by ConorSheehan1** [comment]
Ok I've got a reproducable build.

```Dockerfile
# Dockerfile
FROM ubuntu:16.04

WORKDIR ~/graphql_test

# rails deps
RUN apt-get update && apt-get install -y apt-utils ruby ruby-dev curl \
    libsqlite3-dev bzip2 build-essential libssl-dev libreadline-dev \
    zlib1g-dev nodejs tzdata

# new rails app
RUN gem install rails -v 5.1.4 && rails new .

# use sqlite3 1.3.x, issue with 1.4.0 https://github.com/rails/rails/issues/35153
# tutorial specifies graphiql 1.5.0 so add it
# graphql:install adds graphiql to gemfile, but unpinned, so remove it with sed
RUN bundle remove sqlite3 && bundle add sqlite3 -v 1.3.13 && \
    bundle add graphql -v 1.8.13 && \
    bundle add graphiql-rails -v 1.5.0 --group=development && \
    rails generate graphql:install && \
    sed -i '$ d' Gemfile && \
    bundle install
```

### Step to reproduce
1. Build the container  
```docker build --network=host -t graphql .```
2. Run the container
```docker run -it --network=host graphql```
3. **The issue** DateTimeType and BaseMutations are not defined
```
$ ls app/graphql/mutations/base_mutation.rb
ls: cannot access 'app/graphql/mutations/base_mutation.rb': No such file or directory

$ rails c
Running via Spring preloader in process 43
Loading development environment (Rails 5.1.6.1)
irb(main):001:0> GraphQL::Types              
=> GraphQL::Types
irb(main):002:0> GraphQL::Types::DateTimeType
NameError: uninitialized constant GraphQL::Types::DateTimeType
	from (irb):2
irb(main):014:0> GraphQL::Mutations::BaseMutation
NameError: uninitialized constant GraphQL::Mutations
	from (irb):14
```

### Host info
OS: ubuntu 16.04
Docker version 18.09.1, build 4c52b90

### Container info
OS: ubuntu 16.04
Rails 5.1.6.1
graphql (1.8.13)
graphiql-rails (1.5.0)

**Comment by jonatasbaldin** [comment]
Thanks for all the details @ConorSheehan1! I'm waiting for a review on your PR from a Ruby dev, since I'm not :<

**Comment by RStankov** [comment]
Hey, I opened a PR which fixes this issue 👉 https://github.com/howtographql/howtographql/pull/897

**Comment by jonatasbaldin** [comment]
Fixed at #897.
